src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -104,7 +104,10 @@ export const StatusRoute: FC<StatusProps> = ({ order, match }) => {
                     order={order}
                     useLastSubmittedOffer
                     showOfferNote={isSubmittedOffer}
-                    showCongratulationMessage={order.state === "SUBMITTED"}
+                    showCongratulationMessage={
+                      order.state === "SUBMITTED" &&
+                      order.source !== "private_sale"
+                    }
```

Comment:
this is kinda unnecessary (since a PS order can't be in Submitted state), but added it for extra safety. let me know if you disagree.